blockchain_nodes/parity/home/parity/.local/share/io.parity.ethereum/config.toml

```diff
@@ -0,0 +1,41 @@
+[parity]
+#chain = "./home/parity/.local/share/io.parity.ethereum/chain.json"
+#base_path = "./home/parity/.local/share/io.parity.ethereum"
+#db_path = "./home/parity/.local/share/io.parity.ethereum/chains"
+#keys_path = "./home/parity/.local/share/io.parity.ethereum/keys"
+
+[rpc]
+disable = false
+port = 8545
+interface = "all"
+cors = ["all"]
+hosts = ["all"]
+apis = ["web3", "eth", "net", "parity", "traces", "rpc", "personal", "parity_accounts", "signer", "parity_set"]
+
+[network]
+bootnodes = [
+  "enode://147573f46fe9f5cc38fbe070089a31390baec5dd2827c8f2ef168833e4d0254fbee3969a02c5b9910ea5d5b23d86a6ed5eabcda17cc12007b7d9178b6c697aa5@172.16.0.10:30303",
```

Comment:
What was `172.16.0.10` originally?

Does this have to change now that we are no longer using this inside docker?

Reply:
Mhmm... maybe I can even remove this then... I'll give it a try